ios/Classes/DRM/FLTEzdrmAssetsLoaderDelegate.m

```diff
@@ -0,0 +1,113 @@
+//
+//  FLTEzdrmAssetsLoaderDelegate.m
+//  better_player
+//
+//  Created by Koldo on 18/05/2021.
+//
+
+#import "FLTEzdrmAssetsLoaderDelegate.h"
+
+@implementation FLTEzdrmAssetsLoaderDelegate
+
+NSString *_assetId;
+
+NSString * KEY_SERVER_URL = @"https://fps.ezdrm.com/api/licenses/";
+
+- (instancetype)initWithCertificateUrl:(NSURL *)certificateUrl {
+    self = [super init];
+    _certificateURL = certificateUrl;
+    return self;
+}
+
+- (void)removeKeyWith:(NSFileManager *)fileManager {
```

Comment:
Do we need that method? Seems to be empty.

Reply:
Fixed in this commit 71a3256